src/main/java/com/github/rodionovsasha/commentservice/controllers/UserController.java

```diff
@@ -0,0 +1,24 @@
+package com.github.rodionovsasha.commentservice.controllers;
+
+import com.github.rodionovsasha.commentservice.entities.User;
+import com.github.rodionovsasha.commentservice.services.UserService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import static com.github.rodionovsasha.commentservice.Application.API_BASE_URL;
+import static org.springframework.util.MimeTypeUtils.APPLICATION_JSON_VALUE;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping(API_BASE_URL + "/user")
+public class UserController {
+    private final UserService service;
+
+    @GetMapping(value = "/{id}", produces = APPLICATION_JSON_VALUE)
```

Comment:
Do you really need to define `produces` explicitly?

Reply:
Default:
{}
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/bind/annotation/RequestMapping.html#produces--